### PR TITLE
fixed error log that would cause two log lines to run into each other…

### PR DESCRIPTION
**Pull request by alejandroEsc**
**What this PR does / why we need it**:
Fixed error log that would cause two log lines to run into each other. Logs running into each other can be reproduced by running local-up-cluster.sh on osx.

**Release note**:
```NONE
```

### CODE REVIEW

File: cmd/kubelet/app/server.go [hunk 1]
```diff
@@ -271,7 +271,7 @@ func initKubeletConfigSync(s *options.KubeletServer) (*componentconfig.KubeletCo
 // not be generated.
 func Run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) error {
 	if err := run(s, kubeDeps); err != nil {
-		return fmt.Errorf("failed to run Kubelet: %v", err)
+		return fmt.Errorf("failed to run Kubelet: %v\n", err)
```
Comment (yujuhong) [inline cmd/kubelet/app/server.go]: I don't think it's necessary to end an error message with a newline. What's the issue?

Comment (alejandroEsc) [inline cmd/kubelet/app/server.go]: The issue here is that when this error is reported, and then kubelet restarted, the next error created will be on the same line causing a lot of confusion. I suppose we could start the logs with /n at the beginning but that seems a bit odd to me.

Comment (yujuhong) [inline cmd/kubelet/app/server.go]: You should change where the error is logged/printed instead.

Comment (alejandroEsc) [inline cmd/kubelet/app/server.go]: OK, let me take another look, only place that prints that error already adds a new line to that.

File: cmd/hyperkube/hyperkube.go [hunk 2]
```diff
@@ -129,7 +129,7 @@ func (hk *HyperKube) Run(args []string) error {
 		err := baseFlags.Parse(args)
 		if err != nil || hk.helpFlagVal {
 			if err != nil {
-				hk.Println("Error:", err)
+				hk.Println("Error:", err, "\n")
```
Comment (yujuhong) [inline cmd/hyperkube/hyperkube.go]: `Println` should already include an newline at the end. This is not needed.

File: cmd/hyperkube/hyperkube.go [hunk 3]
```diff
@@ -178,7 +178,7 @@ func (hk *HyperKube) Run(args []string) error {
 
 	err = s.Run(s, s.Flags().Args())
 	if err != nil {
-		hk.Println("Error:", err)
+		hk.Println("Error:", err, "\n")
```
Comment (yujuhong) [inline cmd/hyperkube/hyperkube.go]: Same here. `Println` should already include an newline at the end. This is not needed.

Comment (alejandroEsc) [inline cmd/hyperkube/hyperkube.go]: doh, you sir, are right.

**Comment by alejandroEsc** [comment]
@yujuhong will you merge or is supposed to be done by the submit queue?

**Comment by yujuhong** [comment]
@alejandroEsc the submit queue will do this. You can check the status in http://submit-queue.k8s.io/

**Pull request merged**